README.rst

```diff
@@ -72,3 +83,13 @@ Now hack away! You can run the tests using::
     psql -c 'create database psycopg_test'
     export PSYCOPG_TEST_DSN="dbname=psycopg_test"
     pytest
+
+
+Cross-compiling
+---------------
+
+To use cross-platform zipapps created with `shiv`__ that include Psycopg
+as a dependency you must also have ``libpq`` installed. See
+:ref:`the section above <Hacking>` for install instructions.
```

Comment:
This link doesn't render well: see [here](https://github.com/psycopg/psycopg/blob/d2f6590c7a7f3f58c4207d759f83d1a2b08a8173/README.rst). `:ref:` is a Sphinx role; in pure reST you can just use `Hacking_` to link to a title in the same document.